app/assets/javascripts/all_legacy.js

```diff
@@ -1,18 +1,18 @@
 // This is a manifest file used by the runner for JavaScript unit tests.
```

Comment:
This all file seems to comprise both frontend and admin so it doesn't seem like something that should be legacy - it seems the intention would be to use it for all JS. 

Do we instead need different files used for JS tests? maybe we don't need an `all` file anymore?

Reply:
Only thing I see that is required in the all and not the other files is:

```
//= require govuk-admin-template
```

Reply:
I have reverted this change as you are correct, it isn't legacy.